README.md

```diff
@@ -159,6 +159,63 @@ You'll need to upload these to an s3 bucket, then you can use these files to
 [create a simulation application](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-application.html),
 and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-job.html) in RoboMaker.
 
+
+## Generate map for your world
+
+Requires ROS-Melodic + Gazebo9.
+
+### Pre-build
+
+```bash
+# Install dependencies (Ubuntu >18.04)
+sudo apt-get install libxml-xpath-perl libxml2-utils
+```
+
+```bash
+# Fetch and install ROS dependencies
+cd robot_ws
+rosws update
+rosdep install --from-paths src --ignore-src -r -y
+
+cd ../simulation_ws
+rosws update
+rosdep install --from-paths src --ignore-src -r -y
+cd ..
+```
+
+### Generate Occupancy Map
+
+```bash
```

Comment:
Similarly, can you add a short blurb about what "Generate Occupancy Map" is for?